tests/system/providers/amazon/aws/example_emr.py

```diff
@@ -164,10 +164,14 @@ def get_step_id(step_ids: list):
         task_id="add_steps",
         job_flow_id=create_job_flow.output,
         steps=SPARK_STEPS,
-        wait_for_completion=True,
```

Comment:
I just moved this outside of the example code block (to line 170 below) as done elsewhere.  We've been trying to clean up the snippets that get pasted into the docs but must have missed this one.  No functionality change